cursorless-nx/apps/cursorless-org/pages/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { EmbeddedVideo } from '@cursorless/react/embedded-video';
+import Head from 'next/head';
+import Button from '../components/Button';
+import { TITLE, YOUTUBE_SLUG } from '../components/constants';
+import Social from '../components/Social';
+import { ReactComponent as Logo } from './svg/logo.svg';
+
+export default function LandingPage() {
+  const smallScaling = 'sm:w-smBase sm:h-smBase sm:text-smBase';
+  const stretchedScaling =
```

Comment:
add comment here about how the font size applies to the element containing everything, so that then `em`s get scaled with the whole layout.  should mention we prob want to apply the font size to `html` element instead and then can use `rem`s